FirebaseAuth/Tests/Unit/FIRAuthTests.m

```diff
@@ -1513,6 +1521,39 @@ - (void)testSignInWithCustomTokenSuccess {
   OCMVerifyAll(_mockBackend);
 }
 
+/** @fn testSignInWithCustomTokenWithoutRefreshTokenSuccess
+    @brief Tests the flow of a successful @c signInWithCustomToken:completion: call without a
+   refresh token.
+ */
+- (void)testSignInWithCustomTokenWithoutRefreshTokenSuccess {
+  OCMExpect([_mockBackend verifyCustomToken:[OCMArg any] callback:[OCMArg any]])
+      .andCallBlock2(^(FIRVerifyCustomTokenRequest *_Nullable request,
+                       FIRVerifyCustomTokenResponseCallback callback) {
+        XCTAssertEqualObjects(request.APIKey, kAPIKey);
+        XCTAssertEqualObjects(request.token, kCustomToken);
+        XCTAssertTrue(request.returnSecureToken);
+        dispatch_async(FIRAuthGlobalWorkQueue(), ^() {
+          id mockVeriyCustomTokenResponse = OCMClassMock([FIRVerifyCustomTokenResponse class]);
+          [self stubTokensWithMockResponseNoRefreshToken:mockVeriyCustomTokenResponse];
+          callback(mockVeriyCustomTokenResponse, nil);
+        });
+      });
+  // If refreshToken is not present, GetAccountInfo is not invoked.
```

Comment:
Can you add a verification statement that verifies `GetAccountInfo` is not invoked?

Reply:
Added `OCMReject([_mockBackend getAccountInfo:[OCMArg any] callback:[OCMArg any]]);`